ui/src/pages/backoffice/Item/ItemDetails/components/ItemMetadata/ItemMetadata.js

```diff
@@ -29,6 +34,14 @@ export default class ItemMetadata extends Component {
       },
     ];
 
+    if (itemDetails.metadata.price) {
+      // insert after acquisition pid
```

Comment:
I do not understand this comment, if it is essential a little rephrasing is required, otherwise ... poof!